examples/ex6p.cpp

```diff
@@ -129,7 +129,11 @@ int main(int argc, char *argv[])
    //    the Laplace problem -\Delta u = 1. We don't assemble the discrete
    //    problem yet, this will be done in the main loop.
    ParBilinearForm a(&fespace);
-   if (pa) { a.SetAssemblyLevel(AssemblyLevel::PARTIAL); }
+   if (pa)
+   {
+      a.SetAssemblyLevel(AssemblyLevel::PARTIAL);
+      a.SetDiagonalPolicy(mfem::Operator::DIAG_ONE);
```

Comment:
Do we need the explicit `mfem::` namespace?

Reply:
You are right, we don't need it; I removed it here.